plugins/module_utils/network/vyos/argspec/ospf_interfaces/ospf_interfaces.py

```diff
@@ -71,12 +71,16 @@ def __init__(self, **kwargs):
                         "authentication": {
                             "type": "dict",
                             "options": {
-                                "plaintext_password": {"type": "str"},
+                                "plaintext_password": {
+                                    "type": "str",
+                                    "no_log": True,
+                                },
                                 "md5_key": {
                                     "type": "dict",
+                                    "no_log": True,
```

Comment:
This would cause key_id to be hidden too. Not sure if. `key_id` needs to be masked.

Reply:
Rest LGTM +1.